### ISSUE TITLE
[Bug] Importing from CSV file containing non-ASCII characters causes the application to crash

### ISSUE THREAD

**Issue by fsobolev** [nlogozzo/NickvisionMoney#15]
As I mentioned in #14, I experience crashes when importing from CSV. I found out that it only happens if the CSV file contains non-ASCII characters (cyrillic in my case).

CSV file:
```
ID,Date,Description,Type,RepeatInterval,Amount,Group
1,2022-11-06,Тестовая транзакция,0,0,99,1
2,2022-11-06,Покупка,1,0,12,-1
```
Output:
```
terminate called after throwing an instance of 'std::out_of_range'
  what():  map::at
Aborted (core dumped)
```

**Comment by fsobolev** [comment]
Sorry, I forgot to mention that if after restart I open the account I was importing into, the first transaction is actually imported, but not the second one.

**Comment by nlogozzo** [comment]
Mine doesn't crash, but it only imports the second transaction. 🤔🤔
![image](https://user-images.githubusercontent.com/17648453/200192380-3e01c3c4-680c-4773-9961-c58976e9b1c4.png)

**Comment by fsobolev** [comment]
I see you have Test group. If I add group first, then import CSV, the result is the same as yours.
Crash happens when trying to import to new, absolutely clean account.

By the way, the crash also happens if I try to import CSV that has more transactions than the account. Even for files containing only ASCII-characters (both CSV and .nmoney).
If I try to import CSV that has less transactions than the account, nothing happens (notification shows that 0 transactions imported).

**Comment by nlogozzo** [comment]
Oh yes I need to fix the import and export csv to support importing and exporting groups. Right now only transactions are handled, and if the group doesn't exist, it won't import the transaction.

**Comment by nlogozzo** [comment]
@zothma is working on a fix for the CSV import/export system :) It should be finished soon!

**Comment by nlogozzo** [comment]
Fixed by #39
